### DISCUSSION TITLE
Can we not run in firebase test lab?

### DISCUSSION THREAD

**Discussion by Rohithgilla12** [fzyzcjy/flutter_convenient_test#245] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19389850/183431666-6c566db9-8609-45bd-82b4-1cddfe232d13.png)

According to the logs they state we are capturing the test results properly. Is it a known issue?

**Comment by fzyzcjy** [comment, marked as answer]
It is because we do not use `flutter test` but use `flutter run` deliberately. This is because we need the hot restart functionality to support various use cases.

However, if you do not need them, I guess it should be ok to use `flutter run` and then able to be on firebase.

Btw, if you are not using it in firebase, **just ignore that error!** The results *are* captured properly. That error is just because our hack makes existing sanity check unhappy but there is no actual harm.

**Reply by Rohithgilla12** [reply]
Thanks!

**Reply by fzyzcjy** [reply]
You are welcome :)

**Reply by fzyzcjy** [reply]
Btw you can also ask them in the issue section, it may be easier for others to search and see it
